lib/rubocop/cop/offense.rb

```diff
@@ -63,7 +63,20 @@ class Offense
       attr_reader :corrector
 
       PseudoSourceRange = Struct.new(:line, :column, :source_line, :begin_pos,
-                                     :end_pos)
+                                     :end_pos) do
+        alias_method :first_line, :line
+        alias_method :last_line, :line
+        alias_method :last_column, :column
+
+        def column_range
+          column...last_column
+        end
+
+        def size
+          end_pos - begin_pos
+        end
+        alias_method :length, :size
```

Comment:
[note] This redefines `Struct#size` and `Struct#length`.

See https://ruby-doc.org/core-2.7.1/Struct.html#size-method

Reply:
I think that's ok.